lib/tasks/tariff_importer.rake

```diff
@@ -23,8 +23,8 @@ namespace :importer do
         # We will be fetching updates from Taric and modifying primary keys
         # so unrestrict it for all models.
         Sequel::Model.descendants.each(&:unrestrict_primary_key)
-
-        TaricImporter.new(ENV["TARGET"], ENV["DATE"]).import
+        Sequel::Model.plugin :skip_create_refresh
```

Comment:
Nice: could we link to the documentation http://sequel.jeremyevans.net/rdoc-plugins/classes/Sequel/Plugins/SkipCreateRefresh.html